### ISSUE TITLE
Scribus 1.4.7 doesn't handle grouped items properly?

### ISSUE THREAD

**Issue by kumo** [berteh/ScribusGenerator#112]
Hello, I have had problems with groups in Scribus 1.4.7. If I try to create a single file then visually everything is correct, but the items that should be in groups are all inside the group of the first page, thus the PDF export doesn't show those them.

If I open the file in Scribus 2.5.4 the groups are correct.

Thanks

[before.zip](https://github.com/berteh/ScribusGenerator/files/2195093/before.zip)
[after.zip](https://github.com/berteh/ScribusGenerator/files/2195094/after.zip)

**Comment by berteh** [comment]
Thanks kumo for the issue report. I was planning on dropping support for 1.4 line... as it slowly reaches end of line per https://www.scribus.net/scribus-1-4-7-and-1-5-4-released/

Is this issue really important that I should looke how to fix it for 1.4.7... since it works fine in recent Scribus ?

**Comment by kumo** [comment]
Thanks for the feedback. I have only recently started using Scribus so I wasn't sure which version to be using. Also, I can run Scribus 1.4.7 on my linux laptop and my mac laptop, but 1.5.4 only runs on my mac, however this I shall need to resolve.

Personally, I don't think that a 1.4.7 fix is necessary, especially if I'm the one asking for it.

Thanks for this tool though as I was looking for a cross-platform alternative to the mac app Sketch for generating certificates and Scribus and ScribusGenerator work very well. I did discover though that I had to add my vectors and static text to the master page otherwise the final file was over 100MB compared to 2MB and Scribus was very slow.
